.NET/Microsoft.Recognizers.Text.DateTime/Parsers/BaseTimeZoneParser.cs

```diff
@@ -72,7 +72,9 @@ public static string ConvertMinsToRegularFormat(int offsetMins)
         public static string NormalizeText(string text)
         {
             text = Regex.Replace(text, @"\s+", " ");
-            return Regex.Replace(text, @"timezone", "time");
+            text = Regex.Replace(text, @"timezone$", string.Empty);
+            text = Regex.Replace(text, @"time$", string.Empty);
```

Comment:
All three replaces can be the same regex. It can also be a compiled regex for a little more speed.